### DISCUSSION TITLE
[BUG] Your Issue TitleFailed to connect to 127.0.0.1 port 8443: Connection refused ,,when i try to connect with mirth connect

### DISCUSSION THREAD

**Discussion by RenEpic** [nextgenhealthcare/connect#6151] [Q&A]
**Describe the bug**
A clear and concise description of what the bug is. Is this consistently reproducible?

**To Reproduce**
Setup steps (if required). Example:
1. Setup table in database with this schema
2. Add rows '...'

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happens.

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. Linux (CentOS), Windows 10, MacOS]
 - Java Distribution/Version [e.g. OpenJDK 11, Java 8 (201)]
 - Connect Version [e.g. 3.8.0]

**Workaround(s)**
Are there one or more workarounds for this issue currently?

**Additional context**

**Comment by pacmano1** [comment]
You post doesn't make much sense.
